packages/collections/lib/stats/last.u.jest.js

```diff
@@ -0,0 +1,108 @@
+jest.mock('@orbiting/backend-modules-utils')
+
+describe('const LAST_INTERVALS', () => {
+  const { LAST_INTERVALS: UUT } = require('./last')
+
+  it('is an array', () => {
+    expect(UUT).toBeInstanceOf(Array)
+  })
+
+  it('has at least 1 item', () => {
+    expect(UUT.length).toBeGreaterThanOrEqual(1)
+  })
+
+  it('has objects w/ key and interval prop', () => {
+    UUT.map(o => {
+      expect(o).toHaveProperty('key')
+      expect(o).toHaveProperty('interval')
+    })
+  })
+})
+
+describe('createCache()', () => {
+  const { createCache: UUT } = require('./last')
+
+  const { cache } = require('@orbiting/backend-modules-utils')
+
+  const expectedDefaultOptions = {
+    namespace: 'collections',
+    prefix: 'stats:last',
+    ttl: 60 * 60 * 24,
+    key: 'any'
+  }
```

Comment:
I can't see why it's necessary to test that the resolver uses this cache settings exactly. Why must the tests fail when these params are adjusted in `last.js`?